pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -432,6 +433,12 @@ lib.makeOverridable (
         postInstall = optionalString isModular ''
           mkdir -p $dev
           cp vmlinux $dev/
+
+          mkdir -p $dev/lib/modules/${modDirVersion}/build/scripts/gdb
+          cp -r ../scripts/gdb/* $dev/lib/modules/${modDirVersion}/build/scripts/gdb
+          cp scripts/gdb/linux/constants.py $dev/lib/modules/${modDirVersion}/build/scripts/gdb/linux
+          cp scripts/gdb/linux/.*.cmd $dev/lib/modules/${modDirVersion}/build/scripts/gdb/linux
```

Comment:
`cp -L` wouldn't work to do the whole thing in one go, would it?

Reply:
that work! updated!